translations/ui-users/en.json

```diff
@@ -3,6 +3,13 @@
   "resultCount": "{count, number} {count, plural, one {Record found} other {Records found}}",
   "closedLoansCount": "{count, number} {count, plural, one {closed loan} other {closed loans}}",
 
+  "brd.renewConfirmation": "Renew Confirmation",
+  "brd.header.renewalStatus": "Renewal status",
+  "brd.header.title": "Title",
+  "brd.failedRenewal": "Item not renewed",
+  "brd.successfulRenewal": "Item successfully renewed",
+  "brd.itemNotRenewed": "<strong>{count}</strong> {count, plural, one {item} other {items}} <b>not</b> renewed. ",
```

Comment:
Does this work now? I used to have issues with SafeHTMLMessage rendering `<strong>` tags. I had to switch to `<b>` for everything despite not wanting to 😆

Reply:
@doytch Yup, it works. Translations guy @rjruizes helped me with that.

Reply:
This is good to know! :D